### PR TITLE
Hide pagination when totalPages <= 0

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/124.

This should've been a change in just one place, but our other components haven't used our <SearchableList /> component yet. I will do this refactoring another time.

### CODE REVIEW

File: app/javascript/components/common/SearchableList.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const Results = ({
           setOrder={setOrder}
         />
       ) : null}
-      {latestData ? (
+      {latestData && latestData.meta.totalPages > 0 ? (
```
Comment (iHiD) [inline app/javascript/components/common/SearchableList.tsx]: Should this not be `> 1`? If there's only one result, totalPages is still `1` I think?

Also, can we add a test for this pls.

Comment (ErikSchierboom) [reply app/javascript/components/common/SearchableList.tsx]: I think you're right :)

**Review by iHiD — CHANGES_REQUESTED** [review]
See my inline comment :)

File: app/javascript/components/common/SearchableList.tsx (hunk 1 above)
Comment (kntsoriano) [reply app/javascript/components/common/SearchableList.tsx]: Oh yes!

**Comment by kntsoriano** [comment]
@iHiD I'm opening a new PR based on the new `main` branch.

**Pull request closed without merging**